Packages/CandidateUI/Sources/CandidateUI/HorizontalCandidateController.swift

```diff
@@ -229,13 +243,15 @@ public class HorizontalCandidateController: CandidateController {
         contentRect.size = NSSize(width: 36.0, height: 20.0)
         nextPageButton = NSButton(frame: contentRect)
         nextPageButton.setButtonType(.momentaryLight)
-        nextPageButton.bezelStyle = .smallSquare
-        nextPageButton.title = "»"
+        nextPageButton.bezelStyle = .toolbar
```

Comment:
Turns out this is macOS 14.0+.

Reply:
Oh, thanks for the catch! There was no warning!  :(

Reverted to .smallSquare, which looks the same since isBordered is false anyway.

Reply:
On our end, we'll need to trigger GitHub actions on approved PR going forward…

There doesn't seem to be a way to re-open a merged PR on GitHub, and so I've created a revert-the-revert PR in #385.